internal/datastore/crdb/crdb.go

```diff
@@ -213,7 +213,7 @@ func (cds *crdbDatastore) SnapshotReader(rev datastore.Revision) datastore.Reade
 	}
 
 	querySplitter := common.TupleQuerySplitter{
-		Executor:         pgxcommon.NewPGXExecutor(createTxFunc),
+		Executor:         pgxcommon.NewPGXExecutor(createTxFunc, false),
```

Comment:
This is temporal until we enable caveats in CRDB, because this code is shared between PSQL and CRDB. I plan to remove this as soon as CRDB caveat support lands.